src/api/app/controllers/application_controller.rb

```diff
@@ -456,4 +457,13 @@ def forward_from_backend(path)
   def shutup_rails
     Rails.cache.silence! unless Rails.env.development?
   end
+
+  def set_influxdb_tags
+    anonymous = User.current.present? && !User.current.is_nobody?
```

Comment:
[nitpick] `present?` is not needed

Reply:
aehm, what exactly is anonymous then? I find nobody more anonymous than !nobody (is that body then?)

Reply:
Jaja ... The conditional was wrong, fixed!